rdr_service/offline/export_va_workqueue.py

```diff
@@ -0,0 +1,223 @@
+import os
+import datetime
+
+from rdr_service import clock, config
+from rdr_service.api_util import list_blobs, delete_cloud_file
+from rdr_service.offline.sql_exporter import SqlExporter
+from rdr_service.dao.hpo_dao import HPODao
+from rdr_service.dao.participant_summary_dao import ParticipantSummaryDao
+
+CSV_HEADER = ['PMI ID',
+              'Biobank ID',
+              'Last Name',
+              'First Name',
+              'Middle Initial',
+              'Age Range',
+              'Participant Status',
+              'Primary Consent Status',
```

Comment:
Should the export have the authored dates for the consents and survey completion dates?

Reply:
The dictionary created by ParticipantSummaryDao.to_client_json doesn't include the datetime fields if they're null even when the line to strip None values is skipped. I could include those values by adding lines like participant_filtered.setdefault('consentForStudyEnrollmentAuthored','UNSET') before creating participant_row

Reply:
I think that will be good. We'll want to make sure the export has the same data as the work queue.